free5gc-operator/controllers/free5gc_smfcfg.go

```diff
@@ -0,0 +1,100 @@
+package controllers
+
+var SMFCfgTemplate string = `
+info:
+  version: 1.0.2
+  description: SMF configuration
+
+configuration:
+  serviceNameList:
+    - nsmf-pdusession
+    - nsmf-event-exposure
+    - nsmf-oam
+
+  sbi:
+    scheme: http
+    registerIPv4: smf-nsmf
+    bindingIPv4: 0.0.0.0
+    port: 80
+    tls:
+      key: config/TLS/smf.key
+      pem: config/TLS/smf.pem
+
+  nrfUri: http://nrf-nnrf:8000
+  pfcp:
+    addr: {{ .PFCP_IP }}
+  smfName: SMF
+  snssaiInfos:
+    - sNssai:
+        sst: 1
+        sd: 010203
+      dnnInfos:
+        - dnn: internet
+          dns:
+            ipv4: 8.8.8.8
+    - sNssai:
+        sst: 1
+        sd: 112233
+      dnnInfos:
+        - dnn: internet
+          dns:
+            ipv4: 8.8.8.8
+    - sNssai:
+        sst: 2
+        sd: 112234
+      dnnInfos:
+        - dnn: internet
+          dns:
+            ipv4: 8.8.8.8
+  plmnList:
+    - mcc: "208"
```

Comment:
In our discussions, we talked about a "3gpp config" resource that would contain this information. Shouldn't this be in the SMFDeployment resource?